cmd/aws-application-networking-k8s/main.go

```diff
@@ -128,14 +128,24 @@ func main() {
 		setupLog.Fatal("cloud client setup failed: %s", err)
 	}
 
+	// do not create the webhook server when running locally
+	var webhookServer k8swebhook.Server
+	isLocalDev := config.DevMode != ""
+	if !isLocalDev {
+		webhookServer = k8swebhook.NewServer(k8swebhook.Options{
+			Port:     9443,
+			CertDir:  "/etc/webhook-cert/",
+			CertName: "tls.crt",
+			KeyName:  "tls.key",
+		})
+	}
+
```

Comment:
When using ```make run```, DevMode will be non-empty, so locally running the controller will skip this webhook configuration. Otherwise, the controller will only start if the webhook cert files are present (at ```/etc/webhook-cert/tls.crt``` and ```tls.key```).